homeassistant/components/default_config/manifest.json

```diff
@@ -18,7 +18,8 @@
     "sun",
     "system_health",
     "updater",
-    "zeroconf"
+    "zeroconf",
+    "search"
```

Comment:
Should this be part of `default_config`? I think that it should be part of `frontend`?

Reply:
Even better, then I don't have to check if it is loaded :-)